lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -71,6 +74,8 @@ def import_kymolinegroup_from_csv(filename, image, delimiter=";"):
 
     indices = data[:, 0]
     lines = np.unique(indices)
+
+    image = CalibratedKymographChannel.from_kymo(kymo, channel)
```

Comment:
If this is all it takes to construct a `CalibratedKymographChannel` then the current API feels better? As in: less parameters and concrete types is better than a bunch of parameters that could just as well be mapped to that concrete type.

Reply:
In the future once we've settled on the format for the exported .csv the import function will become public. `CalibratedKymographChannel` will always remain private (it was introduced to be able to handle calibration for kymotracker, but users should never construct this manually).

So in the future, the new API is needed as `Kymo` is public. Implementing now is mostly an improvement for our internal use

Reply:
OK, now the PR makes sense to me: Function will become public but still uses an internal type in its signature, and this needs to be fixed.

Reply:
edited the PR description for clarity